Sources/BraintreeCore/Public/BraintreeCore/BTPaymentMethodNonce.h

```diff
@@ -21,7 +21,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param nonce       A transactable payment method nonce.
  @return A Payment Method Nonce, or `nil` if nonce is nil.
 */
-- (nullable instancetype)initWithNonce:(NSString *)nonce;
+- (nullable instancetype)initWithNonce:(NSString * _Nullable)nonce;
```

Comment:
I believe nullable is the default in obj-c, but want to confirm - does adding these nullability annotations affect existing merchant integrations?

Reply:
We have `NS_ASSUME_NONNULL_BEGIN` at the top of this file, so non-nullable is the default here. But allowing the initializer to accept `null` as well as non-null string shouldn't be a breaking change for merchants.